File: RQ1/CodeForces-Automatic/281A.c
#include <stdio.h>
#include <ctype.h>
#include <assert.h>

int main()
{
    char s[1000];
    scanf("%s", s);
    
    assert(s[0] != '\0'); // PRE: The input word is non-empty
    
    s[0] = toupper(s[0]);
    
    assert(s[0] >= 'A' && s[0] <= 'Z'); // POST: The first letter is capitalized
    
    printf("%s\n", s);
    return 0;
}

File: RQ1/CodeForces-Automatic/281A.py
def main():
    # PRE:
    # Input Format:
    # A single line contains a non-empty word. This word consists of lowercase and uppercase English letters. The length of the word will not exceed 103.
    # POST:
    # Program Description:
    # Capitalization is writing a word with its first letter as a capital letter. Your task is to capitalize the given word.Note, that during capitalization all the letters except the first one remains unchanged.
    # Output Format:
    # Output the given word after capitalization.
    word = input()
    
    new = ''
    
    new += word[0].capitalize() + word[1:]
    
    print(new)
